2022/src/day10.scala

```diff
@@ -0,0 +1,48 @@
+package day10
+
+import locations.Directory.currentDir
+import inputs.Input.loadFileSync
+
+import Direction.*
+
+def INPUT(): String = loadFileSync(s"$currentDir/../input/day10")
```

Comment:
please can you refactor the code so that there exists `def part1(input: String)` and `def part2(input: String)` that receive the input already read from the file, like all the other files in this directory - its necessary for the solver on the webpage

Reply:
I also want to say its not typical in scala to use ALL_CAPS, please refactor to UpperCamelCase

Reply:
I switched to pure function, forwarding the input: String parameter to each function. The signatures are now, I think, as you desired